tests/decode.ts

```diff
@@ -0,0 +1,48 @@
+import { test, assertEquals } from "../deps.ts";
+import { Column, Format } from "../connection.ts";
+import { Oid } from "../oid.ts";
+import { decode } from "../decode.ts";
+
+const encoder = new TextEncoder();
+
+test(function decodeInet() {
+  const column = new Column(
+    "inet_col", // name
+    16410,      // tableOid
+    1, // index
+    Oid.inet, // dataTypeOid
+    -1, // columnLength
+    -1, // typeModifier
+    Format.TEXT // format
+  );
```

Comment:
I'm not really happy with that... you shouldn't create `Column` and fill it with info by hand. I'd much prefer integration test - create a table in test case that consist of these columns, then add a row and finally query the table to check if you get proper values back.

Reply:
@bartlomieju Sure do you want table to be created in constants.ts or ...? Should tests go into decode.ts or do you want something like data_types.ts that would have integration tests for data types?

Reply:
`data_types.ts` sounds good :+1: just create table there for a single test, we can iterate on that later, thanks!